### PR TITLE
Update strings

### PR DESCRIPTION
**Pull request by vlvagerviwager**

### CODE REVIEW

**Comment by vlvagerviwager** [comment]
@tborychowski: Could you please review? 😁

File: index.html [hunk 1]
```diff
@@ -89,7 +89,7 @@
 				<div class="history"><select class="history-list" size="2" tabindex="2"></select></div>
 
 				<a href="#" class="header-btn js-copy ion-md-link" title="Copy link" data-go="copy"></a>
-				<span class="copy-link-confirmation">Copied!</span>
+				<span class="copy-link-confirmation">Link copied!</span>
```
Comment (tborychowski) [inline index.html:92]: for that I'd need to change the css a bit :-)

**Review by tborychowski — APPROVED** [review]
thanks

File: index.html (hunk 1 above)
Comment (vlvagerviwager) [reply index.html:92]: 😮

**Pull request merged**